src/types/UiaTextRangeBase.cpp

```diff
@@ -1330,3 +1178,37 @@ RECT UiaTextRangeBase::_getTerminalRect() const
         gsl::narrow<LONG>(result.top + result.height)
     };
 }
+
+#pragma warning(push)
+#pragma warning(disable : 26447) // compiler isn't filtering throws inside the try/catch
+std::wstring UiaTextRangeBase::_getTextValue() const noexcept

```

Comment:
what is this now?

Reply:
Discussed offline.

This is necessary because we need the `const` on that function header.

Decision: I'll make our current `GetText` use this as a helper function.